sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -78,7 +78,7 @@
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
 # This should be in the beam-[version]-[date] format, date is optional.
```

Comment:
<!--new_thread; commit:87ea0681e36c0640c02c8d137f83ad73b95e1bc3; resolved:0-->
The point of using a dated version string is that incompatibilities should not break users who check out the beam repo.  We should rebuild a new container at the current date instead of using the catch-all dev container.

Reply:
I've updated to a dated container.